exchanges/exchange_types.go

```diff
@@ -207,3 +207,56 @@ type Base struct {
 	Config        *config.ExchangeConfig
 	settingsMutex sync.RWMutex
 }
+
+// url lookup consts
+const (
+	RestSpot URL = iota
```

Comment:
I think we should standardise these for example:
[protocol][asset][functionality]
RestSpot
RestFutures
RestSpotSandBox
etc

Edge cases and chanin analysis can be:
RestSupplementaryOne
RestSupplmentaryTwo
etc.

This is just a generalisation across multiple exchanges.